subcommand/controller/command.go

```diff
@@ -93,10 +100,12 @@ func (c *Command) Run(_ []string) int {
 	}
 
 	if os.Getenv("ENABLE_WEBHOOKS") != "false" {
+		mgr.GetWebhookServer().CertDir = c.flagWebhookTLSCertDir
```

Comment:
Could we add a comment saying that setting cert dir will also watch for file changes and rotate certs?